terraform/terraform.tfvars.example

```diff
@@ -10,3 +10,6 @@ container_command           = ["/bin/bash", "-c", "echo hello && sleep 86400"]
 container_environment_variables = {
   "ASPNETCORE_ENVIRONMENT" = "production"
 }
+key_vault_access_users = [
+  "someone_example.com#EXT#@tenantname.onmicrosoft.com",
```

Comment:
I would like to understand where we get these 'Principle Names' in the documentation.